src/main/webapp/player.html

```diff
@@ -93,13 +93,16 @@ <h3 class="col text-muted">WebRTC Play</h3>
 				</div>
 			</div>
 
-			<p>
-				<button onclick="startPlaying()" class="btn btn-primary"
-					id="start_play_button">Start Playing</button>
-				<button onclick="stopPlaying()" class="btn btn-primary"
-					id="stop_play_button">Stop Playing</button>
+			
+				<div class="form-group">
+					<button onclick="startPlaying()" class="btn btn-primary"
+						id="start_play_button">Start Playing</button>
+					<button onclick="stopPlaying()" class="btn btn-primary"
+						id="stop_play_button">Stop Playing</button>
+				</div>
+				<span class="badge badge-success" id="bitrateInfo" style="font-size:14px;display:none"
```

Comment:
It's a warning message. I think we should use the correct class. (badge-warning)